pom.xml

```diff
@@ -776,5 +786,28 @@ limitations under the License.
         <javadoc-link>https://docs.oracle.com/javase/9/docs/api/</javadoc-link>
       </properties>
     </profile>
+    <profile>
+      <!-- Extra checks that are disabled in the regular build, enabled for
+      releases and on demand. -->
+      <id>pedantic</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.owasp</groupId>
+            <artifactId>dependency-check-maven</artifactId>
+            <configuration>
+              <failBuildOnCVSS>${failBuildOnCVSS}</failBuildOnCVSS>
```

Comment:
Do you need to propagate this into the plugin's configuration? Would certainly be nice if the plugin would just pick it up from the definition in the properties up-above (I don't know off the top of my head if it does this though)

Reply:
Please review https://github.com/julianhyde/calcite-avatica/commit/aabf8b935cb1ceecce4337df782f4463ecefb1bd. I added a suppressions file.